src/gui/FileDialog.cpp

```diff
@@ -62,7 +62,7 @@ QString FileDialog::getSaveFileName(QWidget* parent, const QString& caption, QSt
     }
     else {
         if (dir.isEmpty()) {
-            dir = config()->get("LastDir").toString();
+            dir = config()->get("LastDir").toString().append("/NewDatabase.kdbx");
```

Comment:
getSaveFileName() is not just used for kdbx databases.
This belongs to DatabaseTabWidget::saveDatabaseAs()